dolphinscheduler-common/src/main/resources/remote-logging.properties

```diff
@@ -0,0 +1,36 @@
+# Whether to enable remote logging
```

Comment:
May I ask why not move these configs into `application.yaml`?

Reply:
> May I ask why not move these configs into `application.yaml`?

do you mean application.yaml in api/master/worker?

If so, we bind configuration loader mechanism binding to springboot.
Also, maybe `application.yaml` are stored configuration about spirngboot runtime?

Reply:
Can we use yaml type, e.g. remote-logging.yaml or logging.yaml

```
remote:
    logging:
         enable: false

```

Reply:
sure. will do

Reply:
![image](https://github.com/apache/dolphinscheduler/assets/13224827/9dde3e07-401c-4f6d-9378-69307ee3d715)
I may leverage quartz style yaml organized format like above.